conans/test/integration/toolchains/cmake/cmakedeps/test_cmakedeps.py

```diff
@@ -114,3 +114,42 @@ def package_info(self):
         content = f.read()
         assert 'set(hello_OBJECTS_RELEASE "${hello_PACKAGE_FOLDER_RELEASE}/mycomponent.o")' in content
         assert 'set(hello_hello_say_OBJECTS_RELEASE "${hello_PACKAGE_FOLDER_RELEASE}/mycomponent.o")' in content
+
+
+def test_cpp_info_component_error_aggregate():
+    # https://github.com/conan-io/conan/issues/10176
```

Comment:
Linking the issue and describing the test is very nice, but, when the assert is only kind of "this is working" would be great to describe here why exactly `"consumer/1.0 (test package): Running test()"` was not in the output before the fix without reading the whole issue thread.